PreLoad.cmake

```diff
@@ -58,10 +58,9 @@ execute_process(COMMAND uname -m OUTPUT_VARIABLE ARCH)
 # By default, prefer clang on Linux
 # But note, that you still may change the compiler with -DCMAKE_C_COMPILER/-DCMAKE_CXX_COMPILER.
 if (OS MATCHES "Linux"
+    # some build systems may use CC/CXX env variables
```

Comment:
Preload.cmake sets defaults for (lazy) devs who don't specify C and C++ compilers explicitly via cmd line argument or via environment variable. Instead of weakening the check it would be better to simply specify the compiler explicitly.